### PR TITLE
Entities

### PR DESCRIPTION
**Pull request by DenisRupp**
Add getters and setters. Generated repository, deleted joins.

### CODE REVIEW

File: src/AppBundle/Entity/BaseUser.php [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+ *
+ */
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="user")
+ * @ORM\InheritanceType("SINGLE_TABLE")
+ * @ORM\DiscriminatorColumn(name="type", type="string")
+ * @ORM\DiscriminatorMap({"user" = "User", "personal" = "Personal"})
+ */
+abstract class BaseUser
```
Comment (Etheriq) [inline src/AppBundle/Entity/BaseUser.php:20]: must be implement UserInterface

File: src/AppBundle/Entity/BaseUser.php~ [hunk 2]
```diff
@@ -0,0 +1,151 @@
+<?php
```
Comment (Etheriq) [inline src/AppBundle/Entity/BaseUser.php~:1]: *.~ files must be in .gitignore file

File: src/AppBundle/Entity/Feedback.php [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    /**
+     * @var  /DataTime
+     *
+     * @Gedmo\Timestampable(on="create")
+     * @ORM\Column(name="created_at", type="datetime")
+     */
+    private $createdAt;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="User", inversedBy="feedbacks")
```
Comment (Etheriq) [inline src/AppBundle/Entity/Feedback.php:47]: maybe you mean AppBundle:User  This is Doctrine standard style

File: src/AppBundle/Entity/BaseUser.php (hunk 1 above)
Comment (0TshELn1ck) [inline src/AppBundle/Entity/BaseUser.php:20]: cool double phpdoc

**Pull request closed without merging**